contrib/firebaseio/src/main/java/contrib/JacksonCoder.java

```diff
@@ -0,0 +1,152 @@
+/**
+ * Copyright (c) 2015 Google Inc.
```

Comment:
Following the join library example [here](https://github.com/GoogleCloudPlatform/DataflowJavaSDK/blob/master/contrib/join-library/src/main/java/com/google/cloud/dataflow/contrib/joinlibrary/Join.java#L2), may be more appropriate to put "The Google Cloud Dataflow Firebase Connector authors" eg